src/VisualStudio/Core/Next/Remote/JsonRpcClient.cs

```diff
@@ -34,6 +34,8 @@ internal class JsonRpcClient : IDisposable
             _rpc.Disconnected += OnDisconnected;
         }
 
+        public abstract void Dispose();
```

Comment:
:arrow_right: This form is used when the type has a finalizer *or* the type is not sealed. In this case it is the lack of `sealed` which mandates this form.